### ISSUE TITLE
How can I reproduce the performance of flowtron_ljs.pt?

### ISSUE THREAD

**Issue by jaeseongyou** [NVIDIA/flowtron#17]
How can I reproduce the performance of _**flowtron_ljs.pt**_? Have you trained the model _**flowtron_ljs.pt**_ with the default configuration _**config.json**_? I found the given configuration a bit confusing due to the following oddities:
1) 'train_config: epochs' is set to 10,000,000 (too much)
2) 'train_config: batch_size' is set to 1 (too little)
3) 'model_config: n_flows' is set to 2 (unlike in the paper where 3 is proposed)
4) 'model_config: n_components' is set to 0 (shouldn't the lower bound for this param be 1?)

**Comment by rafaelvalle** [comment]
1. stop training when the loss plateaus or the model starts overfitting
2. in our experiments batch size 1 leads to better generalization error and validation loss
3. that's a typo, thank you for catching that.
4. we only use the gmm setup if n_components is larger than 0.
